docs/dev/building-on-zksync/hello-world.md

```diff
@@ -83,29 +69,35 @@ module.exports = {
   networks: {
     zkSyncTestnet: {
       url: "https://testnet.era.zksync.dev",
-      ethNetwork: "goerli", // Can also be the RPC URL of the network (e.g. `https://goerli.infura.io/v3/<API_KEY>`)
+      ethNetwork: "goerli", // RPC URL of the network (e.g. `https://goerli.infura.io/v3/<API_KEY>`)
       zksync: true,
     },
   },
   solidity: {
-    version: "0.8.17",
+    version: "0.8.8",
```

Comment:
Any reason why we are changing this to `"0.8.8"`? @kailash-manasarovar

Reply:
Yes, it breaks with 0.8.17 -> please recreate the issue, if you can, and report it.